admin/src/scenes/inbox/components/ticketMessage.js

```diff
@@ -29,10 +29,9 @@ export default ({ ticket: propTicket }) => {
     try {
       if (!id) {
         setTicket(undefined);
```

Comment:
il faut return, sinon ca boucle entre "selectionnez un ticket" et le loader
```suggestion
        return setTicket(undefined);
```